Makefile

```diff
@@ -257,7 +257,7 @@ ifeq (,$(shell which opm 2>/dev/null))
 	set -e ;\
 	mkdir -p $(dir $(OPM)) ;\
 	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
-	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.23.0/$${OS}-$${ARCH}-opm ;\
+	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.28.0/$${OS}-$${ARCH}-opm ;\
```

Comment:
Please provide justification for this bump.

Reply:
opm 1.28 matches the current version of operator-sdk

- go version bumped to 1.18 -> 1.19
- k8s bumped 1.24 -> 1.25 -> 1.26
- go libraries bumped
- FBC caching refactored
- fix channel sorting
- improve selinux labeling
- bug fixes